UtopiaRDS/utopiaAirlineSchema.sql

```diff
@@ -19,17 +19,22 @@ CREATE TABLE `tbl_flights` (
   KEY `departure_idx` (`departure`),
   KEY `destination_idx` (`destination`),
   CONSTRAINT `departure` FOREIGN KEY (`departure`) REFERENCES `tbl_airports` (`code`),
-  CONSTRAINT `destination` FOREIGN KEY (`destination`) REFERENCES `tbl_airports` (`code`)
+  CONSTRAINT `destination` FOREIGN KEY (`destination`) REFERENCES `tbl_airports` (`code`),
+  CONSTRAINT `departure!=destination` CHECK (`departure` != `destination`),
+  CONSTRAINT `departure_date<destination_date` CHECK (`departure_date` < `arrival_date`)

```

Comment:
Do you really want to have the constraint name contain special characters?

Reply:
I would agree in general, but this is less important for constraint names than for field names, since our code will never refer to the constraint by name.

Reply:
I agree. And, that is why I made it a suggestion more than anything else.